docs/installing.md

```diff
@@ -0,0 +1,80 @@
+# Running a local Haveno test network
+
+These are the steps needed to setup a local Haveno test network.
```

Comment:
For development, they should use a local monero stagenet.

For the shared test network, we should use the public stagenet.

These instructions need to decide if they're documenting both or one.

Reply:
The point of this setup is to use one single network for testing. We remove the need for a local stagenet and only use our private shared stagenet, to make things easier for the contributor.